Cesium3DTilesSelection/include/Cesium3DTilesSelection/GltfConverters.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Cesium3DTilesSelection/GltfConverterResult.h>
+#include <CesiumGltfReader/GltfReader.h>
+
+#include <gsl/span>
+
+#include <optional>
+#include <string>
+#include <string_view>
+
+namespace Cesium3DTilesSelection {
+class GltfConverters {
+public:
+  using ConverterFun = GltfConverterResult (*)(
+      const gsl::span<const std::byte>& content,
+      const CesiumGltfReader::GltfReaderOptions& options);
```

Comment:
It's mostly because I want to avoid allocation as much as possible and none of the converters use private data internally, so it maybe better to use function pointer. But I don't have a strong opinion on using interface

Reply:
I updated `ConverterFun` to `ConverterFunction` in the mean time and wait for more opinion on the interface